contrib/k8s/examples/cosmosdb/cosmosdb-instance.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: my-cosmosdb-instance
   namespace: default
 spec:
-  clusterServiceClassExternalName: azure-cosmos-document-db
+  clusterServiceClassExternalName: azure-cosmosdb-sql-account
+  clusterServicePlanExternalName: account
```

Comment:
Only question here is whether the file name should maybe be changed.

Reply:
Applicable to other files in this dir also.

Reply:
Let's take care of these in a follow up so we can begin the process of rebasing #347